.github/workflows/witness.yml

```diff
@@ -56,7 +56,7 @@ jobs:
               path: /tmp
 
           - if: ${{ inputs.pre-command != '' && inputs.pull_request == false }}
-            uses: testifysec/witness-run-action@40aa4ef36fc431a37de7c3faebcb66513c03b934
+            uses: in-toto/witness-run-action@40aa4ef36fc431a37de7c3faebcb66513c03b934
```

Comment:
the name of the action remains an issue here, as it's not been moved to the in-toto org.